src/__init__.py

```diff
@@ -0,0 +1,40 @@
+import asyncio
+
+from Animals.Bunny import Bunny
+from Animals.Turtle import Turtle
+from Animals.CompetingAnimal import CompetingAnimal
+from Road import Road
+
+havi = CompetingAnimal("Havi the bunny", 0, 3, 'B')
+tirza = CompetingAnimal("Tirza the turtle", 0, 1, 'T')
+all_animals = [havi, tirza]
+race_road = Road(20)
+race_length = race_road.length_in_steps
+
+
+def race_is_over(animals: list):
+    for animal in all_animals:
+        if animal.has_won(race_length):
+            return True
+    return False
```

Comment:
sounds like you need a variable that indicates if the race is finished instead of the looping the competitors and checking each one of them if their finished (you can also do this way but more efficient)